.github/workflows/tests.yml

```diff
@@ -65,7 +65,7 @@ jobs:
     - name: Run playwright tests
       run: |
           cd ui-tests
-          npx playwright test
+          npx playwright test --update-snapshots
```

Comment:
This should be undone; the test should indeed fail when there are no snapshots as otherwise we are not comparing against a reference.

Reply:
This has been undone.